main/.vuepress/themeConfig/nav.js

```diff
@@ -18,27 +18,27 @@ module.exports = [
       {
         text: 'Before Using Agoric',
         ariaLabel: 'Before Using Agoric Menu',
-        link: '/getting-started/before-using-agoric/',
+        link: '/getting-started/before-using-agoric',
```

Comment:
This difference led to the nav links and the links of the "getting started" page not linking to the same page (the links to the nav linked to outdated content (which shouldn't be in the deployed website, but that's a problem for another time))